src/components/buttonComponent/buttonComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './buttonStyles.css';
+
+interface Props {
+  children?: React.ReactNode;
+  onClick: () => void;
+  className: string;
+}
+
+const Button: React.FC<Props> = (props) => {
```

Comment:
This is fine, just a small note, you could also use restructuring of the props and have something like 
```suggestion
const Button: React.FC<Props> = ({children, onClick}) => {
```
and use that later instead of `props.onClick`etc.

Reply:
I'll look in to it, but I think with props I could manipulate the code a bit better, especially if I'm adding new props, but if you think I should change it, I will 😀